src/cli.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path')
+
+let inquirer = require('inquirer');
+inquirer
+  .prompt({
+   name: 'ruta',
+   message: '¿Que archivo deseas leer',
+       
+  })
+  .then(answers => {
+      let ruta = answers.ruta;
+    route(ruta)
+   
+  })
+  .catch(error => {
+    if(error.isTtyError) {
+      // Prompt couldn't be rendered in the current environment
+    } else {
+      // Something else when wrong
+    }
```

Comment:
El else esta de más porque no se utiliza, en realidad si resuelve ok no necesitas el else
```suggestion
    } 
```